cloudfour/cloudfour.com-patterns
# Chore: Update index.scss to include gutenberg blocks library CSS

## Overview

See #789, and Tyler's suggestion [here](https://github.com/cloudfour/cloudfour.com-patterns/issues/789#issuecomment-642322457). 

In order to remedy a CSS source order loading issue that was causing the WordPress Gutenberg block library CSS from trumping our patterns CSS in Storybook previews, this PR does the following: 

• Creates a new `index-with-wp-block-library` file that imports WP styles before our pattern styles (index.scss)
• Updates `.storybook/preview.js` to include the new `index-with-wp-block-library` instead of `index`
• Removes `@wordpress/block-library/build-style/style.css` from `src/vendor/wordpress/gutenberg.stories.mdx`, previously added in #778 

## Consideration 

With #778 (WP) Gutenberg block styles were only loaded on the Gutenberg stories. With this update, as I understand it, the Gutenberg CSS will now be loaded on every Storybook page. If that is a concern we should explore other alternative approaches. 

## Screenshots

### Before
Notice the secondary button is outlined with a black border and black text. 

![Screen Shot 2020-06-16 at 3 08 54 PM](https://user-images.githubusercontent.com/1427548/84833110-6da22000-afe3-11ea-87af-e105e757cbb6.png)


### After
Here, the secondary button is styled as expected with our `c-button c-button--secondary` styles.

![Screen Shot 2020-06-16 at 2 22 30 PM](https://user-images.githubusercontent.com/1427548/84829661-f8335100-afdc-11ea-8bf7-a1dd282644d4.png)


## Testing

1. View [deploy preview](https://deploy-preview-798--cloudfour-patterns.netlify.app/?path=/story/vendor-wordpress-gutenberg--core-button)
1. Confirm the secondary button is outlined with a blue border and has blue text. 

---

See #789
